pkg/database/databasescriptview_test.go

```diff
@@ -0,0 +1,60 @@
+package database
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/anz-bank/sysl/pkg/parse"
+	"github.com/anz-bank/sysl/pkg/syslutil"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGenerateDatabaseScriptCreate(t *testing.T) {
+	goldenFileName := "db_scripts/postgres-create-script-golden.sql"
+	modelParser := parse.NewParser()
+	mod, _, err := parse.LoadAndGetDefaultApp("database/db_scripts/dataForSqlScriptOrg.sysl",
```

Comment:
Same as before, check if the slashes are ok.

Reply:
fixed